buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/performance/PerformanceTestPlugin.kt

```diff
@@ -266,7 +264,7 @@ class PerformanceTestPlugin : Plugin<Project> {
         }
         create("distributedFlakinessDetection", DistributedPerformanceTest::class) {
             (options as JUnitOptions).excludeCategories(performanceExperimentCategory)
-            reportGeneratorClass = "org.gradle.performance.results.FlakinessReportGenerator"
+            performanceReporter.reportGeneratorClass = "org.gradle.performance.results.FlakinessReportGenerator"
```

Comment:
We don't re-run the flakiness detection performance test. Does this mean we won't get a report?
Same goes for the historical performance test.

Reply:
Fixed.